packages/components/rollup.config.js

```diff
@@ -65,9 +65,12 @@ export default {
     "lodash/groupBy",
     "lodash/omit",
     "lodash/sortBy",
+    "lodash/noop",
+    "lodash/isEmpty",
```

Comment:
I wonder if these + the 2 above should just be
```suggestion
    new RegExp("lodash/.*"),
```